test/contrasts.jl

```diff
@@ -5,11 +5,36 @@ using DataTables
 using CategoricalArrays
 using StatsModels
 
+import StatsModels: ContrastsMatrix
```

Comment:
Hm. It doesn't look like it works without the `import`; the tests fail with `UndefVarError: ContratsMatrix not defined`